engines/director/cast.cpp

```diff
@@ -1276,8 +1275,27 @@ void Cast::loadScriptText(Common::SeekableReadStreamEndian &stream, uint16 id) {
 	if (ConfMan.getBool("dump_scripts"))
 		dumpScript(script.c_str(), kMovieScript, id);
 
-	if (script.contains("\nmenu:") || script.hasPrefix("menu:"))
-		return;
+	// Walk the string to find the first macro definition
```

Comment:
Actually, the relevant code sits in lingo-preprocessor.cpp. See the code starting from line 170.

It was initially brought in ecf09c214c2538ee8419704b9caac1eaca5cbb50

The problem with the current approach besides sitting in a wrong place is that it shifts the line numbers that is not convenient for the error reporting